packages/api/amplify_api/example/integration_test/graphql/api_key_test.dart

```diff
@@ -164,6 +165,38 @@ void main({bool useExistingTestUser = false}) {
             await expectLater(dataCompleter2.future, completes);
           });
 
+          testWidgets('should support where clause when using Model helpers',
+              (WidgetTester tester) async {
+            final blogName =
+                'Integration Test Blog - subscription filter ${uuid()}';
+
+            final postTitle =
+                'Integration Test Post - subscription filter ${uuid()}';
+
+            final blog = await addBlog(blogName);
+
+            // TODO(equartey): Replace this statement when Model object supports parent id field.
+            final pred =
```

Comment:
question: Can this be `Post.BLOG.eq(blog.id)`?. We do that in a test for query and it converts correctly (some unit tests as well). If it does not convert correctly would be good to understand.